supabase-replicator/supabase/seed.sql

```diff
@@ -680,12 +681,10 @@ create or replace function search_contracts_by_group_slugs(group_slugs text[], l
 as $$
 select array_agg(data) from (
     select data
-    from contracts,
-      jsonb_array_elements(data -> 'groupSlugs')
-          as elem
-    where elem ?| group_slugs
+    from contracts
+    where data->'groupSlugs' ?| elem
     and is_valid_contract(data)
-    order by (to_jsonb(data) ->> 'uniqueBettors7Days')::int desc, to_jsonb(data) ->> 'slug'
+    order by (data->'uniqueBettors7Days')::int desc, data->'slug'
```

Comment:
I thought we had to use `to_jsonb` in order to use unindexed columns in the json blob. Although looking at `user_events` I see that was mistaken. I have to use `to_jsonb` in the queries in the sql editor though I thought...

Reply:
I wrote some stuff in Discord explaining this. Basically, you only need to use `to_jsonb` in queries if the index is on `to_jsonb` stuff.

However, it's basically a moot point here, because this ordering isn't important for an index to be serving. The important part of the query is finding the contracts that are in the specific groups -- that's the part that needs an index in order to be fast.